source/guides/recipes/macos

```diff
@@ -0,0 +1,27 @@
+# Nix(OS) development on macOS
+
+Over the recent years, the Nix(OS) community have made great strides in adding support for macOS. Today, it is possible to use Macs for comfortable Nix(OS) development. 
+
+# Nix on macOS
+
+First, you need to install Nix on your macOS. Using the [Official Installer](https://nixos.org/download) is a good start. If you need help, follow https://nixcademy.com/posts/nix-on-macos/.
```

Comment:
I would recommend using the Determinate Systems' Nix installer (without the `--determinate` flag) as it supports uninstallation

We recommend that or Lix's installer (a fork of DetSys's) in the nix-darwin README: https://github.com/LnL7/nix-darwin?tab=readme-ov-file#prerequisites